build.gradle

```diff
@@ -88,7 +89,7 @@ subprojects {
   }
 
   test {
-    maxHeapSize = "512m"
+    maxHeapSize = "1024m"
```

Comment:
i reverted it back to 512m and checked in the changes.  However this causes 16 tests to fail with the following error:

Caused by: java.lang.IllegalArgumentException: An attempt was made to allocate more off-heap memory than the JVM can allow. The limit on off-heap memory size is given by the -XX:MaxDirectMemorySize command (or equivalent).
	at org.terracotta.offheapstore.paging.UpfrontAllocatingPageSource.allocateBackingBuffers(UpfrontAllocatingPageSource.java:468)
	at org.terracotta.offheapstore.paging.UpfrontAllocatingPageSource.<init>(UpfrontAllocatingPageSource.java:136)
	at org.terracotta.offheapstore.paging.UpfrontAllocatingPageSource.<init>(UpfrontAllocatingPageSource.java:119)
	at org.ehcache.clustered.server.EhcacheStateServiceImpl$ResourcePageSource.<init>(EhcacheStateServiceImpl.java:368)
	at org.ehcache.clustered.server.EhcacheStateServiceImpl$ResourcePageSource.<init>(EhcacheStateServiceImpl.java:359)
	at org.ehcache.clustered.server.EhcacheStateServiceImpl.createPageSource(EhcacheStateServiceImpl.java:222)